### PR TITLE
fix(release): remove fastlane and iOS process

### PR DESCRIPTION
**Pull request by daniellacosse**
This PR:
- removes fastlane
- removes all apple CI processes that depended on fastlane
- moves the root apple folder to src/cordova/apple (hopefully that's okay 😬)

### CODE REVIEW

File: .github/workflows/build_and_test_debug.yml [hunk 1]
```diff
@@ -90,59 +90,6 @@ jobs:
           path: build/dist/Outline-Client.exe
           if-no-files-found: error
 
-  mac-debug-build:
```
Comment (fortuna) [inline .github/workflows/build_and_test_debug.yml:93]: Please restore macOS. It's fixed by https://github.com/Jigsaw-Code/outline-client/pull/1256

**Comment by daniellacosse** [comment]
@fortuna i see you fixed the macos build, so i'll put that back momentarily

File: package-lock.json [hunk 2]
```diff
@@ -8663,23 +8663,55 @@
       "dev": true
     },
     "node_modules/cordova-osx": {
-      "version": "7.0.0-dev",
-      "resolved": "git+ssh://git@github.com/apache/cordova-osx.git#9b1480d20d6626c0f5a32294e002b87a0cb35c0a",
+      "version": "7.1.0-dev",
+      "resolved": "git+ssh://git@github.com/apache/cordova-osx.git#16b56e4a9297c5195a06a57450aeb983e7053184",
```
Comment (fortuna) [inline package-lock.json:8667]: Why is this changing? You will need to revert this or the build will break.

File: .github/workflows/build_and_test_debug.yml [hunk 3]
```diff
@@ -90,59 +90,6 @@ jobs:
...
-      - name: Install NPM Dependencies
-        run: npm ci
-
-      - name: Set XCode Version
-        run: sudo xcode-select -switch /Applications/Xcode_13.2.app
-
-      - name: Build OSX Client
-        run: npm run action gulp build osx
-
-  ios-debug-build:
```
Comment (fortuna) [inline .github/workflows/build_and_test_debug.yml:117]: Add a TODO to re-add iOS.

File: package-lock.json (hunk 2 above)
Comment (daniellacosse) [reply package-lock.json:8667]: mm good catch

File: .github/workflows/build_and_test_debug.yml (hunk 1 above)
Comment (daniellacosse) [reply .github/workflows/build_and_test_debug.yml:93]: Just did

File: package-lock.json (hunk 2 above)
Comment (daniellacosse) [reply package-lock.json:8667]: I don't even know why this changed, i'll revert the file

**Review by fortuna — APPROVED** [review]

**Pull request merged**